pkg/trace/pb/tracer_payload.proto

```diff
@@ -0,0 +1,25 @@
+syntax = "proto3";
+
+package pb;
+
+import "span.proto";
+
+message TraceChunk {
+	int32 priority = 1;
+	string origin = 2;
+	repeated Span spans = 3;
+	map<string, string> tags = 4;
+	bool droppedTrace = 5;
+}
+
+message TracerPayload {
```

Comment:
When you're done with these, can we please write documentation for both of these structures and their fields? It will be helpful when people jump to definition to see what they are, as they will be added in the generated code too.